aspnetcore-webapi/HideEndpointInSwagger/HideEndpointInSwaggerTests/WeatherForecastControllerTests.cs

```diff
@@ -0,0 +1,67 @@
+using HideEndpointInSwagger.Controllers;
+using Microsoft.Extensions.Logging.Abstractions;
+using Xunit;
+
+namespace HideEndpointInSwaggerTests
+{
+    public class WeatherForecastControllerTests
+    {
+
+        private readonly WeatherForecastController _weatherForecastController;
+
+        public WeatherForecastControllerTests()
+        {
+            _weatherForecastController = new WeatherForecastController(new NullLogger<WeatherForecastController>());
+        }
```

Comment:
Since we will remove the Ilogger, we can change this and inline it as well : 
```suggestion
        private readonly WeatherForecastController _weatherForecastController = new WeatherForecastController();
```